src/sentry/static/sentry/app/publicPath.jsx

```diff
@@ -0,0 +1,12 @@
+/* eslint no-native-reassign:0 */
+/* global __webpack_public_path__ */
+
+/**
+ * Set the webpack public path at runtime. The __sentryGlobalStaticPrefix will
+ * be declared in layout.html.
+ *
+ * NOTE: This MUST be loaded before any other modules in the entrypoint.
```

Comment:
But its after the bootstrap imports.

Reply:
Updated the comment. It can be after bootstrap, just needs to be before other app modules load.